src/main/resources/templates/admin/organization.html

```diff
@@ -1,12 +1,97 @@
 <!DOCTYPE HTML>
 <html xmlns:th="http://www.thymeleaf.org">
 <head th:replace="/admin/header :: head"/>
-<script th:fragment="additionalScript">
+<script th:fragment="additionalScript" th:inline="javascript">
+    /*<![CDATA[*/
     function warnOnDelete(event) {
         if (!confirm("Deleting a report will also delete all its photos and cannot be undone. Are you sure?")) {
             event.preventDefault();
         }
     }
+
+    function loadCss(url) {
+        const link = document.createElement('link');
+        link.href = url;
+        link.rel = 'stylesheet';
+        document.head.appendChild(link);
+    }
+
+    function loadScript(url, func) {
+        const script = document.createElement('script');
+        script.src = url;
+        script.async = false;
+        if (func) {
+            script.onload = func;
+        }
+
+        document.head.appendChild(script);
+    }
+
+    // Shows the map control for defining a test planting site. We defer loading the Mapbox CSS and
+    // JS until it's actually needed.
+    function showMap() {
+        const boundaryInput = document.getElementById('boundary');
+        const mapButton = document.getElementById('showMap');
+        const mapContainer = document.getElementById('map');
+        const mapInstructions = document.getElementById('mapInstructions');
+        const mapSubmit = document.getElementById('mapSubmit');
+
+        loadCss('https://api.mapbox.com/mapbox-gl-js/v2.14.1/mapbox-gl.css');
+        loadCss('https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-draw/v1.4.0/mapbox-gl-draw.css');
+        loadScript('https://api.mapbox.com/mapbox-gl-js/v2.14.1/mapbox-gl.js');
+        loadScript('https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-draw/v1.4.0/mapbox-gl-draw.js', () => {
+            mapButton.style.display = 'none';
+            mapContainer.style.display = 'block';
+            mapInstructions.style.display = 'block';
+
+            mapboxgl.accessToken = /*[[${mapboxToken}]]*/;
+
+            const map = new mapboxgl.Map({
+                container: 'map',
+                fitBoundsOptions: { padding: 10 },
+                style: 'mapbox://styles/mapbox/satellite-streets-v12',
+            });
+
+            const draw = new MapboxDraw({
+                controls: {
+                    polygon: true,
+                    trash: true,
```

Comment:
this allows user to delete drawing and startover? Is there a way for the draw control to label the points with coords info (or maybe we can do that above the button)?

Reply:
saw the trash icon in the screenshot, answers my first question.

Reply:
Maybe the GeoJSON textarea will be good enough for the "see what the coordinates are" use case; about to update this PR to make the geometry data visible and editable.